### PR TITLE
Updated bullet points section

### PR DESCRIPTION
**Pull request by mishimi**

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: doc/process/documentation-guidelines.md [hunk 1]
```diff
@@ -102,20 +102,22 @@ A support person should be able to say "Did you complete step 7?" instead of "Di
...
+### End Lists Correctly
 
-Sentences within bullet points or numbered lists should end in normal punctuation. The sentence or fragment at the end of a bullet point should not have a period. 
+Punctuate bullet points and numbered lists if they end in a full sentence. If they end in a fragment, do not add punctuation.
 
 ##### Correct
 
 ----
-- This is a sentence within a bullet point. This is the end of a bullet point without a period
+- This bullet point ends with period.
```
Comment (jwilander) [inline doc/process/documentation-guidelines.md]: Should be "ends with a period."

**Comment by crspeller** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**